llvm/include/llvm/CodeGen/CodeGenPassBuilder.h

```diff
@@ -1179,6 +1179,8 @@ void CodeGenPassBuilder<Derived>::addOptimizedRegAlloc(
     AddMachinePass &addPass) const {
   addPass(DetectDeadLanesPass());
 
+  addPass(InitUndefPass())
```

Comment:
If suggest running git-clang-format.

Reply:
I have already run this locally and will be updated in the next patch set